lib/bounce_email.rb

```diff
@@ -219,11 +219,13 @@ def check_if_bounce(mail)
     end
 
     def get_original_mail(mail) #worked alright for me, for sure this has to be extended
+      original =
       if mail.multipart?
```

Comment:
Let's try to [adhere to the Ruby style guide here](https://github.com/bbatsov/ruby-style-guide#indent-conditional-assignment) and indent the `if` block to the right to make it more obvious that we're using its return value. Maybe go with the second version, with just one level of indentation.